### DISCUSSION TITLE
how to enable 100tick rate Counter-strike source.

### DISCUSSION THREAD

**Discussion by WahzooAnon** [GameServerManagers/LinuxGSM#3867] [General]
I got a question for the GSM community now i'm running a scoutzknivez Counter-strike source server but in order to get the configure settings for scoutzknivez to work the server needs to be 100tic rate for strafing does anyone know how or could guide me to a some kind of documents would be helpful

**Comment by h3o66** [comment]
You can check this and set instead 100 instead of 128 tick in the conifgs:
https://docs.linuxgsm.com/game-servers/counter-strike-global-offensive#setting-for-a-128-tick-server
